spec.html

```diff
@@ -12996,7 +12996,7 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if BodyText of |RegularExpressionLiteral| cannot be recognized using the goal symbol |Pattern| of the ECMAScript RegExp grammar specified in <emu-xref href="#sec-patterns"></emu-xref>.
           </li>
           <li>
-            It is a Syntax Error if FlagText of |RegularExpressionLiteral| contains any code points other than `"g"`, `"i"`, `"m"`, `"s"`, `"u"`, or `"y"`, or if it contains the same code point more than once.
+            It is a Syntax Error if FlagText of |RegularExpressionLiteral| contains any code points other than `g`, `i`, `m`, `s`, `u`, or `y`, or if it contains the same code point more than once.
```

Comment:
These are code points, not code units. (So the status quo is kind of wrong as well, but this PR probably shouldn't propagate that.)

Reply:
Agh, right, I am conflating the two (though in my defense, it's not just me—compare lines 25051 and 25076 here 😅). Is your vote to just worry about the code _point_ cases and let (sequences of) code units be represented as strings?

Reply:
Yup, that'd be my vote.